demo/e2e/app.e2e-spec.ts

```diff
@@ -0,0 +1,14 @@
+import { DemoPage } from './app.po';
+
+describe('demo App', () => {
+  let page: DemoPage;
+
+  beforeEach(() => {
+    page = new DemoPage();
+  });
+
+  it('should display message saying app works', () => {
```

Comment:
It is not hard to e2e the docs page and a bit complex to test each example if we want to.